docs/DRONECAN.md

```diff
@@ -0,0 +1,38 @@
+# mLRS Documentation: DroneCAN for ArduPilot Systems #
+
+([back to main page](../README.md))
+
+DroneCAN can be utilized for MAVLink and RC with mLRS instead of traditional serial communication if both the flight controller and receiver have the hardware necessary to support it.
+
+## Wiring
```

Comment:
maybe in anticipation of future "Wiring and Powering"

Reply:
Done